src/accounts-service-access.vala

```diff
@@ -201,6 +198,7 @@ public class AccountsServiceAccess : Object
 
     private async void sync_last_running_player_to_accountsservice (string last_running_player)
     {
+#if HAS_LOMIRI_SOUND_SCHEMA
         if (_user_proxy == null)
```

Comment:
-> ifdef HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS

Reply:
Ignoring, as we agreed to simplify this in the next PR.